agent/tcs/handler/handler.go

```diff
@@ -144,6 +151,18 @@ func ackPublishMetricHandler(timer *time.Timer) func(*ecstcs.AckPublishMetric) {
 	}
 }
 
+// anyMessageHandler handles any server message. Any server message means the
+// connection is active
+func anyMessageHandler(client wsclient.ClientServer) func(interface{}) {
+	return func(interface{}) {
+		seelog.Trace("TCS activity occured")
```

Comment:
Is the `Trace` level here intentionally?

Reply:
The similar ACS log line is `Debug` level.

Reply:
TCS channel is more chatty, which means we'll see this printed every 20 seconds if this is changed to `Debug`. I could get rid of this altogether if you have a strong preference here. But, somehow this Trace soothes my mind.